setup/setup_all.sh

```diff
@@ -55,7 +58,10 @@ grant_storage_iam() {
 
 # Link billing account to the current project.
 link_billing_account() {
-  export BILLING_ACCOUNT=$(gcloud beta billing accounts list --format "value(name)" | head -n 1)
+  if [[ "$BILLING_ACCOUNT" ==  "" ]]; then
+    export BILLING_ACCOUNT=$(gcloud beta billing accounts list --format "value(name)" | head -n 1)
```

Comment:
We should ask user for billing account as there may be multiple and first one may not be correct

Reply:
Yes, it's possible. We'll add more details in the README in later PR.